### DISCUSSION TITLE
New to Github.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#134017] [New to GitHub]
How to  join as a member to an organization in Github?

**Comment by deconasser** [comment]
Owner inviting you

**Reply by ghost** [reply]
How will the Owner invite?Is that anything related to badges you get?

**Reply by deconasser** [reply]
What do you mean? 
1. To join an organization on GitHub, you typically need to be invited by an existing member with the appropriate permissions. Here's a step-by-step guide on how this process usually works:

2. Receive an Invitation:

An owner or admin of the organization will send you an invitation to join the organization. This invitation is typically sent to your GitHub email address.
Check Your Email:

3. Open the email you received from GitHub. It should contain a link to accept the invitation.
Accept the Invitation:

4. Click on the link provided in the invitation email. This will take you to GitHub where you can accept the invitation.
Confirm Membership:

5. Once you've accepted the invitation, you may be asked to confirm your membership. Follow any additional prompts to complete the process.
Access Organization Resources:

After accepting the invitation, you'll have access to the organization's repositories, projects, and other resources according to the permissions granted to you. (source: github)

**Comment by Piero1456** [comment]
To join an organization on GitHub, an administrator of the organization needs to invite you. The admin will go to the organization's page, navigate to Settings, then People, and send an invitation to your GitHub username or email. Once invited, you will receive a notification via email or on GitHub itself. Accept the invitation from your email or GitHub notifications, and you will become a member of the organization.

**Comment by ghost** [comment]
No,I didn't receive any invitation link from Organization but I am in the
DISCORD server.What is the DISCORD for?

**Comment by Faiaz3k** [comment]
for that admin or owner need to send invitation ,
ask for that to admin and chake ur mail if get invitation in your mail click the invitation link and you will get member in organisation

**Comment by ghost** [comment]
I tried messaging in DISCORD and even mailing them but I don't get responses.It's tutorial DISCORD so I get only response to the tutorial related doubts not off topic.
